tests/chaos/oadp/conftest.py

```
@@ -79,3 +82,243 @@ def drain_vm_source_node(rhel_vm_with_dv_running, oadp_backup_in_progress):
+            if phase:
+                LOGGER.info(f"Restore {restore.name} entered phase {phase}")
+                break
+
+        return restore
+
+    yield _start_restore
+
+    # Cleanup
+    for restore in created_restores:
+        try:
+            restore.__exit__(None, None, None)  # noqa: FCN001
```

Ruff (0.14.10)

Unused `noqa` directive (unknown: `FCN001`)

Remove unused `noqa` directive

(RUF100)